crates/rnote-engine/src/strokes/vectorimage.rs

```diff
@@ -1,10 +1,12 @@
 // Imports
 use super::content::GeneratedContentImages;
+use super::resize::{calculate_resize_ratio, ImageSizeOption};
 use super::{Content, Stroke};
 use crate::document::Format;
 use crate::engine::import::{PdfImportPageSpacing, PdfImportPrefs};
 use crate::{render, Drawable};
 use kurbo::Shape;
+use na;
```

Comment:
The existing codebase always uses `na::` directly for nalgebra types instead of importing it.